APE0.rst

```diff
@@ -0,0 +1,393 @@
+requirements laid out in the Membership section above. Both self-nominations and
+nominations by others are allowed. The Voting Members are expected to make their
+decisions based on the candidate’s adherence to the membership criteria, above.
+The name of the nominee will be known to Voting Members but will not be shared
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will put it to a vote among active Voting Members. Votes for adding
+  voting members will be carried out at most twice a year to avoid too many
+  elections. The voting dates will be announced with at least four weeks notice,
+  and additional nominations will be accepted until two weeks before the
+  election date. The voting will be open for at least two weeks. To successfully
+  gain an appointment as a Voting Member, the nominee must receive at least four
+  positive votes, as long as that constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+At least one round of voting for adding voting members should be carried out
+before the initial election of the Coordination Committee.
+
+Term and Active/Emeritus Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as *emeritus*. Those who have
+not made any significant contribution for two years may be asked to move
+themselves to the *emeritus* category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to *emeritus*. To record and honor their contributions,
+*emeritus* Voting Members will continue to be listed. *Emeritus* Voting Members
+are not able to participate in votes.
+
+Removing Voting Members
+^^^^^^^^^^^^^^^^^^^^^^^
+In exceptional circumstances, it may be necessary to remove someone from the
+Voting Members against their will. A vote must be held to remove a Voting
+Member. Such a vote is triggered by a motion made by an active Voting Member,
+which must be seconded by an additional Voting Member. The vote must conclude
+no more than three months after the motion is seconded. Removal requires
+approval by two-thirds of all active Voting Members at the time the motion is
+made. The motion, second, and vote will be held in private. Removal under this
```

Comment:
What exactly does "in private" mean? What transparency/accountability is there, eg. should the Ombudsperson audit the result, if even the voting members don't see it? It's hard for me to reword this without knowing more about the intention.

Reply:
This is also along the lines of implementation details where the WG did not want to specify all the details. By default that means it falls to the CoCo + Ombudsperson, but my thinking would be that once we've established the voting software we're using, we'd see if it can also handle secret nominations. If not, we might have to use some other automated process.  But the intent is to not write too much procedure here since it might change over time.  The key element is that it's "private" as in "secret ballot".  So perhaps this is the better wording?:

```suggestion
made. The motion, second, and vote will be by secret ballot. Removal under this
```

Reply:
(actually since this is the last thing holding the PR up and it's mostly a clarification item, I'm going to make the command decision to just accept this and proceed)

Reply:
Looks good to me, that is what was meant anyway